### ISSUE TITLE
Remove baseIRI from n-Quads generation

### ISSUE THREAD

**Issue by gaurav** [phyloref/klados#231]
Because of https://github.com/phyloref/phyx.js/issues/113, we need to include a baseIRI when generating n-Quads:

https://github.com/phyloref/klados/blob/2e338c1746c5136b55cb559f5319d696486e7803/src/components/sidebar/Sidebar.vue#L445

This issue a reminder to remove this once we fix this in phyx.js.

**Comment by gaurav** [comment]
Need to review.

**Comment by hlapp** [comment]
> Protege insists on only reading files that contain full IRIs, which is why a base IRI is required.

Should we document this somewhere so that in future we and others can remember why this is there?

**Comment by gaurav** [comment]
I just retested this, and both the current WebProtege and Protege 5.6.5 appear to support relative URLs now! Given a relative IRI of `#Alligatoridae`, WebProtege turns that into `urn:default:baseUri:#Alligatoridae` while Protege turns that into `file:/Users/gaurav/Downloads/Alligatoridae_Alligatorinae_and_4_others.owl#Alligatoridae`. I'll test this more thoroughly, and once I'm sure it works (and after fixing https://github.com/phyloref/phyx.js/issues/113, if it hasn't been fixed properly), I'll get rid of this base IRI entirely.

**Comment by gaurav** [comment]
There is still a problem here: despite having relative URLs (e.g. `#phyloref1`) as the identifiers (`@id`) for each phyloref, removing the base IRI causes all the identifiers to become blank nodes, which _cannot_ be read by Protege. I think the issue is the JSON-LD library we've incorporated into phyx.js, which we should be able to modify to generate relative URLs correctly.

**Comment by gaurav** [comment]
I don't think we can fix this: the [n-Quads specification](https://www.w3.org/TR/n-quads/#sec-iri) says that "IRIs may be written only as absolute IRIs", and Protege cannot read n-Quads files that use blank nodes to describe OWL individuals. We could write something hacky in which we write out the n-Quads file with a base IRI, and then we go back in and delete the base IRI to change it back into relative IRIs (which appears to work in Protege), but this would violate the n-Quads spec. Going back to a previous suggestion, I'm going to leave a note in the source code explaining why we did this and linking back to this ticket. We can change this behavior in the future if needed.

Two other things we could potentially do:
1. Recommend or require that ID fields in Klados be absolute IRIs.
2. Allow the default base IRI to be changed in Klados.
   - But then the question becomes: should this persist through sessions (by writing it into a cookie)? Should we write this into the Phyx file?

But I think the simplest option is the best one here.
